ML-Transformers/BackgroundEnchantments/copyToDocs.js

```diff
@@ -0,0 +1,11 @@
+const ejs = require('ejs');
+const fs   = require('fs');
+const packageInfo = require('./package.json');
+
+const currentVersion = packageInfo.dependencies['@vonage/ml-transformers']
+
+const folderName = '../../docs/versions/' + currentVersion;
```

Comment:
Would it be possible to keep/map the same structure of folders we have for the code in the repo? I mean code lives at https://github.com/Vonage/vonage-media-transformers-samples/ML-Transformers/BackgroundEnchantments so I am asking whether folder name can be:

```js
const folderName = '../../docs/ML-Transformers/BackgroundEnchantments/versions/' + currentVersion;
```

That way all this gets deployed under https://vonage.github.io/vonage-media-transformers-samples/ML-Transformers/BackgroundEnchantments.

The reason I ask for this is because we might want to deploy more items (not only bits under ML-Transformers/BackgroundEnchantments) in this repo this way in the future.

Reply:
done